lib/spack/spack/build_environment.py

```diff
@@ -240,6 +244,7 @@ def set_module_variables_for_package(pkg, module):
     # TODO: make these build deps that can be installed if not found.
     m.make  = MakeExecutable('make', jobs)
     m.gmake = MakeExecutable('gmake', jobs)
+    m.scons = MakeExecutable('scons', jobs)
```

Comment:
This is necessary to build scons packages. Hopefully it doesn't crash Spack if a user doesn't have scons installed and it's not a dependency...